tsconfig.json

```diff
@@ -16,6 +16,7 @@
         "noUnusedParameters": true,
         "sourceMap": true,
         "strict": true,
+        "strictNullChecks": true,
```

Comment:
This is covered by the [`strict`](https://www.typescriptlang.org/tsconfig#strict) option.

Reply:
Done!